### PR TITLE
test: test_wallet_send_0_eth added

### PR DESCRIPTION
**Pull request by Valentina1133**
Test run on CI: https://ci.status.im/job/status-desktop/job/e2e/job/manual/1194/
Test case: https://ethstatus.testrail.net/index.php?/cases/view/704527

### CODE REVIEW

**Comment by anastasiyaig** [comment]
@Valentina1133 can u please resolve conflicts and we can see how tests will run again? right now new test fails
https://ci.status.im/job/status-desktop/job/qa-automation/job/prs/job/PR-364/4/allure/#suites/488390fb4e988a76ba9e8ecf6f765d5c/fb811334420350fe/

**Comment by Valentina1133** [comment]
> @Valentina1133 can u please resolve conflicts and we can see how tests will run again? right now new test fails
> https://ci.status.im/job/status-desktop/job/qa-automation/job/prs/job/PR-364/4/allure/#suites/488390fb4e988a76ba9e8ecf6f765d5c/fb811334420350fe/

Resolved. Let's see. Interesting that it passed when I ran only this test on ci

**Comment by anastasiyaig** [comment]
i really suspect the flaky locator

```
LookupError: Object '{container={container={name='mainWindow' type='StatusWindow'} type='Overlay' unnamed='1' visible='true'} occurrence='2' type='TextEdit' unnamed='1' visible='true'}' not found. Could not match properties:
    type for object name: '{container={container={name='mainWindow' type='StatusWindow'} type='Overlay' unnamed='1' visible='true'} occurrence='2' type='TextEdit' unnamed='1' visible='true'}'
```

why not using object names?

**Comment by Valentina1133** [comment]
> i really suspect the flaky locator
> 
> ```
> LookupError: Object '{container={container={name='mainWindow' type='StatusWindow'} type='Overlay' unnamed='1' visible='true'} occurrence='2' type='TextEdit' unnamed='1' visible='true'}' not found. Could not match properties:
>     type for object name: '{container={container={name='mainWindow' type='StatusWindow'} type='Overlay' unnamed='1' visible='true'} occurrence='2' type='TextEdit' unnamed='1' visible='true'}'
> ```
> 
> why not using object names?

I changed container to o_SearchBoxWithRightIcon and it helped, this new test passed

File: configs/squish.py [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 AUT_PORT = 61500 + int(os.getenv('BUILD_NUMBER', 0))
 SERVER_PORT = 4322 + int(os.getenv('BUILD_NUMBER', 0))
-CURSOR_ANIMATION = False
+CURSOR_ANIMATION = True
```
Comment (borismelnik) [inline configs/squish.py:5]: Is it really needed?

**Review by borismelnik — APPROVED** [review]
LGTM :)

File: configs/squish.py (hunk 1 above)
Comment (Valentina1133) [reply configs/squish.py]: Yes, you are right, just forgot to remove this change. Changed back to False

**Comment by anastasiyaig** [comment]
@Valentina1133 please resolve conflicts here and merge

**Comment by Valentina1133** [comment]
> @Valentina1133 please resolve conflicts here and merge

Ok. Will wait until checks are completed after resolving conflicts and merge

**Pull request merged**